spec/controllers/trainees/check_details_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Trainees::CheckDetailsController do
+  let(:user) { build(:user) }
+  let(:trainee_policy) { instance_double(TraineePolicy, show?: true, edit?: true) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(user)
+    allow(TraineePolicy).to receive(:new).with(user, trainee).and_return(trainee_policy)
```

Comment:
I don't think you don't need this.

If you do it a case of faking the relationship between `user` => `provider` <= `trainee`

```ruby
      let(:trainee) { create(:trainee, :submitted_for_trn, provider: user.provider) }
```

Reply:
Ah yes - copy pasta from another spec 👌 - removing!